tests/empty_struct.rs

```diff
@@ -0,0 +1,15 @@
+#[macro_use] extern crate custom_derive;
+#[macro_use] extern crate derive_builder;
+
+custom_derive!{
+    /// This is a doc comment for the struct
+    #[warn(missing_docs)]
+    #[allow(non_snake_case)]
+    #[derive(Debug, PartialEq, Default, Builder)]
+    struct IgnoreEmptyStruct {  }
```

Comment:
FYI: The other empty struct syntax (`struct IgnoreEmptyStruct;`) does not work. Do we want to fix this? It's basically just another no-op case in the macro.